central/cve/service/singleton.go

```diff
@@ -16,15 +14,8 @@ var (
 )
 
 func initialize() {
-	var imageCVEDataStore cveDataStore.DataStore
-	if features.PostgresDatastore.Enabled() {
-		imageCVEDataStore = cveDataStore.Singleton()
-	} else {
-		imageCVEDataStore = legacyImageCVEDataStore.Singleton()
-	}
-
-	// TODO: Attach other CVE stores.
-	as = New(imageCVEDataStore, dackbox.GetIndexQueue(), vulReqMgr.Singleton())
+	// This service is not used in postgres.
+	as = New(legacyImageCVEDataStore.Singleton(), dackbox.GetIndexQueue(), vulReqMgr.Singleton())
```

Comment:
Can we remove the call to GetIndexQueue as this will load even if postgres is enabled

Reply:
The init of this service is gated behind postgres flag. I will scrub for other instances of `GetIndexQueue`.